filebeat/module/elasticsearch/audit/config/audit.yml

```diff
@@ -6,5 +6,4 @@ paths:
 exclude_files: [".gz$"]
 
 processors:
-# Locale for timezone is only needed in non-json logs
-- add_locale.when.not.regexp.message: "^{"
+- add_locale: ~
```

Comment:
We add this processor for JSON logs as well because some versions of ES (e.g. 7.1.1) generate JSON audit logs with the `@timestamp` field which does not contain a time zone.